src/overlays/actors/ovl_En_Kusa/z_en_kusa.h

```diff
@@ -8,12 +8,18 @@ struct EnKusa;
 
 typedef void (*EnKusaActionFunc)(struct EnKusa*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ ENKUSA_TYPE_0,
+    /* 1 */ ENKUSA_TYPE_1,
+    /* 2 */ ENKUSA_TYPE_2
+} EnKusaType;
+
 typedef struct EnKusa {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ EnKusaActionFunc actionFunc;
     /* 0x0150 */ ColliderCylinder collider;
     /* 0x019C */ s16 timer;
```

Comment:
```suggestion
    /* 0x019C */ s16 regrowTimer;
```
Since this timer is used in reference to when the bush regrows, would specifying what it times be useful?

Reply:
It gets reset in the generic SetupAction function and ticks up in the generic update function, so I wanted to call it `timerSinceLastActionChange` but that's too long 😅
I feel like the fact it appears in non-regrow code warrants not naming it after the regrow mechanic but idk

Reply:
sounds like a general purpose timer to me, which means the name is just fine imo

Reply:
can add a `// gets reset every action change` next to the var possibly

Reply:
Looking closer, I can see what you mean, helped clear things up thanks :D